src/main/resources/static/password-reset/reset.css

```
@@ -0,0 +1,215 @@
+  border-top: 1px dashed #e2e8f0;
+  font-size: 13px;
+}
+
+.detail-key {
+  font-weight: 700;
+  color: #475569;
+}
+
+.detail-value {
+  text-align: right;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)